platform-sdk/platform-apps/tests/PlatformTestingTool/build.gradle.kts

```diff
@@ -15,31 +15,32 @@
  */
 
 plugins {
-  id("com.swirlds.platform.conventions")
-  id("com.swirlds.platform.application")
-  id("com.swirlds.platform.protobuf")
+    id("com.swirlds.platform.conventions")
+    id("com.swirlds.platform.application")
```

Comment:
Are these formatting changes intentional?

Reply:
I assume that files were never touched after we introduced the new formatting and spotless.